adaptors/satusehat.md

```diff
@@ -8,7 +8,84 @@ title: Satusehat Adaptor
 is a health data exchange ecosystem that connects information systems or
 applications from all members of the Indonesian digital health ecosystem
 including health facilities, regulators, guarantors, and digital service
-providers
+providers.
+
+SATUSEHAT uses HL7 [FHIR](https://www.hl7.org/fhir/) (Fast Healthcare Interoperability Resources) in implementing standard data models and Application Programming Interfaces (APIs).
+
+:::info FHIR
+
+FHIR (Fast Healthcare Interoperability Resources) is a global (international) standard that defines the data format and its elements (called " resources ") and an application programming interface (API ) standard for information exchange
+
+:::
+
+Here is an example of how a POST HTTP request for creating an `Encounter` resource looks like in Satusehat with the FHIR format:
```

Comment:
Well, ok, but this is just an example of a post body. Can we nest it into a post() operation?

Can I also check that this is fake data?